devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -205,17 +201,18 @@ def configure() {
 	// enrolls with default periodic reporting until newer 5 min interval is confirmed
 	sendEvent(name: "checkInterval", value: 2 * 10 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
 
-	// OnOff minReportTime 0 seconds, maxReportTime 5 min. Reporting interval if no activity
-	refresh()
+	zigbee.setColorTemperature(6500) +
```

Comment:
The reason this isn't working like I thought it should is the `setColorTemperature` command is being generated in the DTH rather than being sent to the hub. 

I think an easy way to fix this is change 6500 to 5000 (4000 works too). These are nice even numbers and the original value is retained when converting to and from mireds.

Reply:
@tpmanley Did You mean that zigbee.setColorTemperature(6500) should work like You thought if it was wrapped in sendHubCommand() ?

Should we try sendHubCommand(zigbee.setColorTemperature(6500)) separately or use zigbee.setColorTemperature(5000) instead ?

Reply:
`sendHubCommand` might work but there is also a timing issue in that the device record isn't synced to the hub when `configure` starts executing. So you would need to make sure the command isn't sent to the hub until a few seconds after `configure` starts executing. This is why I suggested just changing the code to `zigbee.setColorTemperature(5000)` because then it doesn't rely on the hub generating the command and avoids the timing issue.